src/components/PlaylistWrapper/PlaylistSelector.tsx

```diff
@@ -34,20 +34,25 @@ const useStyles = makeStyles({
   }
 })
 
+let playlistCache: Playlist[] = [];
+
 export default function (props: { onSelect: (playlist: Playlist) => void }) {
   const classes = useStyles()
 
   async function handleRefresh () {
     setLoading(true)
-    let res = await getPlaylists()
-    setPlaylists(res)
+    setPlaylists((playlistCache = await getPlaylists()))
```

Comment:
Surprised the compiler doesnt complain about using setPlaylists before defining it but otherwise lgtm

Reply:
Compiler go brrrr

@z5160345 
It's wrapped in a function call so it won't execute immediately.
I think it _would_ alert us if I call `handleRefresh()` prior to initialising `setPlaylists`